src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+{{% /notice %}}
+
+## Use Cases
+
+- Request or update device state configuration
+  - _I want to track and control smart home devices from my mobile application_
+  - _I want to remotely control household heaters by turning them on/off or by setting up desired temperature_ 
+- Request device actions based on some commands
+  - _I want to remotely unlock a door for a family visitor_ 
+  - _I want to remotely instruct a device for some action, basis of a command_
+
+## Reference Architecture
+
+This section describes architecture considerations for command and control using the AWS IoT Core device shadow
+ - When requesting update to device state configuration
+ - When requesting device actions based on commands  
+
+{{< tabs groupId="MQTT-arch">}}
+{{% tab name="Device Configuration" %}}
+
+# ![Track and control device state configuration using shadow ](architecture1.svg)
+
+
+- _AWS IoT Core_ is the MQTT message broker processing messages
+- _Device_ is the IoT thing to be controlled
+- _Client Application_ is the remote logic that issues commands, update device state configuration and consumes device telemetry data
+- _Device Shadow_ is the replica which makes a device state available to applications and other services
+- _ShadowTopicPrefix_ is an abbreviated form of the topic, referred to `$aws/things/device1/shadow`(classic shadow topic)
+
+1. The _Device_ establishes an MQTT connection to the _AWS IoT Core_ endpoint and then subscribes to the reserved [MQTT shadow topics](https://docs.aws.amazon.com/iot/latest/developerguide/reserved-topics.html#reserved-topics-shadow) to get and update shadow event operations. Moreover, instead of subscribing individually to each reserved shadow topic, wildcard(_#_) could be used for generalized subscription like _`ShadowTopicPrefix/get/#`_ , _`ShadowTopicPrefix/update/#`_.
+
+    _`ShadowTopicPrefix/get/accepted`_   
+    _`ShadowTopicPrefix/get/rejected`_  
+    _`ShadowTopicPrefix/update/accepted`_    
+    _`ShadowTopicPrefix/update/rejected`_   
+    _`ShadowTopicPrefix/update/delta`_  
+
+2. After successful connect and topic subscription, the _Device_ publish request to _`ShadowTopicPrefix/get`_ topic and process the latest shadow file received on the _`ShadowTopicPrefix/get/accepted`_ topic. After processing configuration changes from _desired_ data in shadow file, _Device_ makes publish request to _`ShadowTopicPrefix/update`_ topic with device latest configuration as _reported_ data on shadow file.     
```

Comment:
What is "shadow file"? Shadow document is how it's referenced in the documentation. Please search/replace for other instances.